src/views/AppRoot/container.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import { isLoggedSelector } from 'Store/concepts/session/selectors'
+
+import { checkLoggedStatus } from 'Store/concepts/session/actions'
+import AppRootComponent from './component'
+
+class AppRoot extends Component {
+  // eslint-disable-next-line react/static-property-placement
+  static propTypes = {
+    isLogged: PropTypes.bool.isRequired,
+    checkLogged: PropTypes.func.isRequired
+  }
+
+  componentDidMount() {
+    const { checkLogged } = this.props
+
+    checkLogged()
+  }
```

Comment:
possibly this should be inside `getInitialProps`

Reply:
This is application without next.js.

Is it network request? If yes you should add loader until get response